src/schema/v2/CollectorProfile/collectorProfile.ts

```diff
@@ -58,20 +59,25 @@ export const CollectorProfileFields: GraphQLFieldConfigMap<
     resolve: ({ self_reported_purchases }) => self_reported_purchases,
   },
   userInterests: {
+    deprecationReason: 'Use "owner#interestsConnection" field instead.',
     type: new GraphQLNonNull(new GraphQLList(userInterestType)),
     resolve: (_collectorProfile, _args, { meUserInterestsLoader }) => {
       return meUserInterestsLoader?.().then(({ body }) => body)
     },
   },
-
-  // moved InquirerCollectorProfileFields here
   location: { type: myLocationType },
   artsyUserSince: dateFormatter(({ artsy_user_since }) => artsy_user_since),
   ownerID: {
```

Comment:
should we deprecate this then?

Reply:
In theory... Though, I've been finding them useful sometimes where you might want to defer loading of some sub-object to the client.